dspace/config/modules/authentication-ldap.cfg

```diff
@@ -167,3 +167,6 @@ autoregister = true
 #login.groupmap.2 = ldap-dept2:dspace-groupA
 #login.groupmap.3 = ldap-dept3:dspace-groupA
 
+# Enables support for StartTLS (default is false). If this flag is true be sure provider_url looks like:
+# ldap://ldap.myu.edu:389
+#starttls=true
```

Comment:
With the merger of #1104, this config will need to be renamed to `authentication-ldap.starttls`